### PR TITLE
Update server values

### PR DESCRIPTION
**Pull request by Kelvin-M**
It may be better to have this in the values.yaml instead of default ! What do you think

### CODE REVIEW

File: charts/substra-backend/templates/deployment-server.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
         {{- end }}
         command: ["/bin/bash"]
         {{- if eq .Values.backend.settings "prod" }}
-        args: ["-c", "uwsgi --http :{{ .Values.backend.service.port }} --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.prod "]
+        args: ["-c", "uwsgi --http :{{ .Values.backend.service.port }} --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 8 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.prod "]
```
Comment (ClementGautier) [inline charts/substra-backend/templates/deployment-server.yaml]: Could be nice to make that a template value so we can tune it on each use case.

File: charts/substra-backend/templates/deployment-server.yaml [hunk 2]
```diff
@@ -98,7 +98,7 @@ spec:
               - name: Accept
                 value: "text/html;version=0.0, */*;version=0.0"
           initialDelaySeconds: 60
-          timeoutSeconds: 5
+          timeoutSeconds: 10
```
Comment (ClementGautier) [inline charts/substra-backend/templates/deployment-server.yaml]: to be reverted?

Comment (Kelvin-M) [reply charts/substra-backend/templates/deployment-server.yaml]: Yes :)

**Comment by Kelvin-M** [comment]
@samlesu @inalgnu  can you review :) ?

**Pull request merged**